docs/source/customize.rst

```diff
@@ -34,6 +34,13 @@ Input Options
 
   Same as above, but with several files of correctly spelled words.
 
+``spelling_word_list_filename='spelling_wordlist.txt,another_list.txt'``
```

Comment:
Since there are merge conflicts to resolve, you could move the paragraph describing the command line option behavior up to the section that talks about the option being a string. You could say something like "The string may include multiple filenames, separated by commas. This is useful when calling Sphinx with ..."